src/app/screens/Home/utils.js

```diff
@@ -0,0 +1,39 @@
+import * as constants from './constants';
+
+export const canAddDot = sentence => {
+  const groups = sentence.split(/[+]|[-]|[x]|[÷]/);
+  return !groups[groups.length - 1].includes(constants.DOT);
+};
+
+export const isDot = character => character === '.';
+
+export const isKeyboardOperator = character =>
+  character === constants.ADDITION ||
+  character === constants.SUBTRACTION ||
+  character === constants.KEYBOARD_MULTIPLICATION ||
+  character === constants.KEYBOARD_DIVISION;
```

Comment:
this is ok, but just in case you didn't know, you can do this at the top of the file

`const { ADDITION, SUBTRACTION, KEYBOARD_MULTIPLICATION, KEYBOARD_DIVISION } = constants;`

so you don't have to call on `constants` every time

Reply:
also, so you don't have to compare for each case, you can write the func like this 

```
export const isKeyboardOperator = character => 
    [ADDITION, SUBTRACTION, KEYBOARD_MULTIPLICATION, KEYBOARD_DIVISION]
    .includes(character)

```

Reply:
great, done!